README.md

```diff
@@ -131,7 +131,7 @@ configured and running on all machines in the cluster before triggering
 Finally we need to set the Ceph container image path:
 
 ```
-/containers/images/ceph set docker.io/ceph/daemon-base:latest
+/containers/images/ceph set docker.io/ceph/ceph:v15.2.2
```

Comment:
How are users going to know which tag to use? (When installing and updating RPM packages, users are not required to specify a version number...)

When a SES7 maintenance update is released, will the users be expected to specify the exact version number of the maintenance update container image to update the system to it?

Reply:
that's the plan

Reply:
@asettle